text/chapter6.md

```diff
@@ -45,7 +47,7 @@ instance showBoolean :: Show Boolean where
   show false = "false"
 ```
 
-This code declares a type class instance called `showBoolean` - in PureScript, type class instances are named to aid the readability of the generated JavaScript. We say that the `Boolean` type _belongs to the `Show` type class_.
+This code declares a type class instance called `showBoolean` - in PureScript, type class instances can be named to aid the readability of the generated JavaScript. We say that the `Boolean` type _belongs to the `Show` type class_.  
```

Comment:
Let's eliminate all manually-named instances. Opened #379 to track. We can come back and delete this line as part of that effort.

Reply:
Also, was whitespace accidentally added to the end of this line?

Reply:
Yes, good catch!
```suggestion
This code declares a type class instance called `showBoolean` - in PureScript, type class instances can be named to aid the readability of the generated JavaScript. We say that the `Boolean` type _belongs to the `Show` type class_.
```